packages/client/src/rpc/modules/engine.ts

```diff
@@ -1226,11 +1226,9 @@ export class Engine {
       }
       this.service.txPool.removeNewBlockTxs(blocks)
 
-      const isPrevSynced = this.chain.config.synchronized
-      this.config.updateSynchronizedState(headBlock.header)
```

Comment:
Should this line be removed? :thinking:

Reply:
Yes.

Reply:
It's was not triggering anything anyhow and is now replaced by a direct setting of `synchronized` on FCU.

Testing eventual side effects right now by running a full Holesky sync (seems nothing so far).

Point of all this is that the tx pool never started.

Reply:
I don't think we should remove this line. The txpool thing is fine. Now each time FCU is called it will set state to synchronized. There is an interval in `sync.ts` which calls it also, which might override it back to syncing. I think this has undesired side effects..?

Reply:
I guess i"m not 100% clear on what this `synchronized` state represents then.  Does it represent the current head that the CL knows of or is it the current canonical head that the `client` is aware of or yet again, is it the current executed head of the client?

cc @g11tech

Reply:
i will lookinto fhis and make a fix shortly, as discussed with @holgerd77 the new logic might open the txpool without us being in the "synchronized" state

Reply:
added the fix:
![image](https://github.com/ethereumjs/ethereumjs-monorepo/assets/76567250/4f17963e-d525-4efc-84c4-10e362b82c2e)